packages/nocodb/src/services/public-datas.service.ts

```diff
@@ -307,7 +307,9 @@ export class PublicDatasService {
 
     for (const file of param.files || []) {
       // remove `_` prefix and `[]` suffix
-      const fieldName = file?.fieldname?.replace(/^_|\[\d*]$/g, '');
+      const fieldName = Buffer.from(file?.fieldname || '', 'binary')
+        .toString('utf-8')
+        .replace(/^_|\[\d*]$/g, '');
 
```

Comment:
Even `base.title` and `model.title` can contain different language characters so let's handle that as well.

Reply:
Those seem to be already using `utf-8` the request body is also properly decoded. It is only the `files` field that is not working.